docs/addons/konveyor.md

```diff
@@ -0,0 +1,213 @@
+# Konveyor Add-On for Amazon EKS Blueprints
+
+Streamline the modernization of your application portfolio to Kubernetes. The Konveyor tool gives you insight throughout the migration process - whether you're making decisions at the portfolio or application level.
+
+Feature set
+
+- Konveyor Hub: Central interface from where you manage your application portfolio and integrate with other Konveyor tools.
+
+- Categorize and group applications by different dimensions (pre-packaged or custom) aligned with technical criteria or your organization structure.
+
+- Assess applications containerization suitablity and risks assessment.
+
+- Assign priority, assess estimated migration effort, and define optimal migration strategy for individual applications.
+
+- Evaluate required changes for Java applications containerization through automated analysis (pre-packaged or custom rules).
+
```

Comment:
Architecture diagram on integration with EKS helps customers to relate it better.